OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -40,6 +40,9 @@ public class AircraftInfo : ITraitInfo, IPositionableInfo, IFacingInfo, IMoveInf
 
 		public readonly int TurnSpeed = 255;
 
+		[Desc("Turn speed to apply when aircraft flies in circles while idle. Defaults to TurnSpeed if negative.")]
+		public readonly int IdleTurnSpeed = -1;
```

Comment:
Is there any feasible way to split this and e4691a0ca12bc2435dc5da03268194c664dbfd69 into their own prereq PR?

Reply:
#15848